.github/workflows/php.yml

```diff
@@ -1,55 +1,52 @@
+---
 name: Unit tests
 
 on:
   push:
+    paths:
+      - '**.php'
+      - 'composer.json'
     branches:
-      - 'master'
-    tags:
       - '*'
+      - '!v1'
+      - '!v2'
   pull_request:
+    paths:
+      - '**.php'
+      - 'composer.json'
+    branches:
+      - '*'
+      - '!v1'
+      - '!v2'
+  workflow_dispatch:
 
 jobs:
   build:
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        php: ['7.2', '7.3', '7.4']
-        phpunit: ['8.0', '9.0']
-        composer-arg: ['']
-        include:
-          - php: '8.0'
-            phpunit: '9.0'
-            composer-arg: 'ignore-platform-req=php'
-        exclude:
-          - php: '8.0'
-            phpunit: '8.0'
-          - php: '7.2'
-            phpunit: '9.0'
-    runs-on: ubuntu-latest
+        php: [ '7.2', '7.3', '7.4', '8.0' ]
     continue-on-error: ${{ matrix.php == '8.0' }}
-    name: PHP ${{ matrix.php }}, PHPUnit ${{ matrix.phpunit }}
-    
+    name: PHP ${{ matrix.php }}
```

Comment:
Since this library not just _uses_ PHPUnit as dependency, but is in itself also intended as an addon for PHPUnit, I'd like to keep testing against multiple versions of PHPUnit (at least those that are supported by this lib, which tend to correlate with the officially supported versions).

Reply:
The idea is, to let composer decide, which PHPUnit version to use, depending on the current PHP version. It should cover most versions, without explicitly declaring them.

Reply:
My assumption was that most users will have a dependency to `phpunit/phpunit` declared directly in the `require-dev` section of their `composer.json`, and will then require-dev any assertion libraries (like this one) on top/in addition to their existing PHPUnit requirements.